test/e2e/support/world.coffee

```diff
@@ -29,13 +30,15 @@ World = (callback) ->
   # Generate a configuration file and save it to path.
   @writeConfigFile = (dir, file, done) =>
     mkdirp dir, 0o0755, (err) =>
-      throw new Error(err) if err
+      return done err if err
```

Comment:
It calls the callback with the error message.  It's what should have been done the whole time.